.gitignore

```diff
@@ -17,6 +17,7 @@
 
 # Ignore all config files
 /config/*.yml
+/config/aws_ips.json
```

Comment:
No, I figured we'd compile it into the docker container image so that it's at least consistent for all instances (and so that we can't have AWS's site be down when we're starting. Either it's there, or we docker's build fails, was my philosophy.